### PR TITLE
Same Slug Archive Dismissible

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#39617](https://central.tri.be/issues/39617)

### CODE REVIEW

File: src/resources/js/events-admin.js [hunk 1]
```diff
@@ -693,3 +693,33 @@ jQuery( document ).ready( function( $ ) {
 		return false;
 	} );
 } );
+
+
+/**
+ * Notice Dismiss
+ */
+( function( $ ) {
```
Comment (barryhughes) [inline src/resources/js/events-admin.js]: Does it make sense to have this is common?

Comment (bordoni) [inline src/resources/js/events-admin.js]: I like that idea. I will change it up.

**Comment by bordoni** [comment]
Updated.

**Comment by borkweb** [comment]
:+1:

**Pull request merged**